_templates/component/new/index.ejs.t

```diff
@@ -0,0 +1,12 @@
+---
+to: packages/asc-ui/src/components/<%= parent %>/<%=name%>/index.ts
+---
+import <%=name%>Style from './<%=name%>Style'
+export { default as <%=name%> } from './<%=name%>'
+
+const <%=name%>Styles = {
+  <%=name%>Style,
+}
+export {
+  <%=name%>Styles
```

Comment:
Other components export their styled counterpart as `...Style` instead of `...Styles`. Change of approach?

Reply:
Mistake, I'll remove this :)